quickstack/manifests/heat_controller.pp

```diff
@@ -66,22 +72,28 @@
   class { '::heat::api_cloudwatch':
       enabled => str2bool_i("$heat_cloudwatch"),
   }
-
+  
+  if $heat_use_ssl {
+    $protocol = "https"
```

Comment:
you need to use str2bool_i($heat_use_ssl). Otherwise, it will consider the variable to be string and will result in True for both 'true' and 'false'.